crates/elements/src/_docs/attributes/font_family.md

```diff
@@ -1,12 +1,14 @@
-### font_family
+With the `font_family` you can specify what font you want to use for the inner text.
 
-With the `font_family` you can specify what font do you want to use for the inner text.
+Check out the [custom font example](https://github.com/marc2332/freya/blob/main/examples/custom_font.rs)
+to see how you can load your own fonts.
 
-Limitation: Only fonts installed in the system are supported for now.
+<!-- TODO: Example of checking if a font exists with skia_safe -->

```

Comment:
> TODO: Example of checking if a font exists with skia_safe

What do you mean by this?

Reply:
I removed the "limitation" because it's not true since you can load your own fonts.

As for the todo, I thought it makes sense to include a way to check if a font exists, rather than just "hope" that it exists. Like if you use "Inter" as a font, you might want to know if it exists, and fallback to another font.

Reply:
A priority list for `font_family` could be implemented, just like in CSS, what do you think?

Reply:
Do you mean Serif, Sans-serif, Monospace, Cursive and Fantasy? That would be nice yeah.